tests/test_model_v2_VariableDeclaration.py

```diff
@@ -0,0 +1,74 @@
+"""Tests of OData Model: class VariableDeclaration"""
+
+import pytest
+from pyodata.v2.model import VariableDeclaration, Types
+from pyodata.exceptions import PyODataException
+
+
```

Comment:
While it is good to have so many unit test, since the problem is "nullable is incorrectly parsed from metadata" I would consider adding to fixtures new XML with metadata for this case - small, few fields, just every nullable type present.

It may be a bit more readable from one test that is parsing real metadata where the problem is when it fails.

Reply:
> nullable is incorrectly parsed from metadata

Metadata parsing was fixed in https://github.com/SAP/python-pyodata/commit/f9144b318b455534cbf2f14a0511c33bd3592c0a

> I would consider adding to fixtures new XML with metadata for this case - small, few fields, just every nullable type present.

Sure, I am happy to review your PR with such a test.

Reply:
I am OK that we just agree on that - sure different PR.